doc/manage-duniverse.mld

```diff
@@ -0,0 +1,212 @@
+{1 Considerations}
+
+When using opam-monorepo the question arises: how to manage it in comparison to
+the regular opam workflow? Compared to regular opam usage using opam-monorepo
+brings lockfiles and a duniverse folder into the repository. There are multiple
+ways to handle this, with multiple advantages and disadvantages. This document
+explains the considerations behind dealing with each of those, along with our
+recommendations.
+
+The document mentions Git for source control as it currently is the most
+popular choice, but similar considerations are valid for most other version
+control systems as well.
+
+{1 Methods}
+
+We assume the opam file is committed to the repository. Running [opam monorepo
+lock] will create a [.locked] opam file in addition to the original opam file.
+Calling [opam monorepo pull] will create a [duniverse] folder into which it
+will unpack the sources of the packages that are mentioned in the lockfile.
+
+{2:lockfile-in-git Lockfile in Git}
+
+Another approach is very similar to the regular opam workflow, but in addition
+to having the [.opam] file committed to the repository, also the lockfile that
+[opam monorepo lock] generates is added.
+
+Advantages include:
+
+{ul
+{- The project repository only grows by one file, thus easy to adopt}
+{- The packages to be installed are reproducible between different machines
```

Comment:
This items and the next one are advantages tied to the use of opam-monorepo itself and are common to the two described workflows.

We can probably remove them from here, eventually moving them elsewhere in the manual.